$OpenBSD: patch-vhdlparser_CMakeLists_txt,v 1.1 2019/06/14 20:20:54 rsadowski Exp $

https://github.com/doxygen/doxygen/commit/058e1a54ee5e8bd8a60c85af5ff1df68d87da3da#diff-b30a837e9b79290121abe5c5cf69c6bb

"Generate configvalues.h only as a dependency for the VHDL parser. Fixes a
problem in combination with Ninja v1.9.0 because of duplicate rules for
generating configvalues.h."

Index: vhdlparser/CMakeLists.txt
--- vhdlparser/CMakeLists.txt.orig
+++ vhdlparser/CMakeLists.txt
@@ -1,14 +1,6 @@
-# configvalues.h
-add_custom_command(
-    COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/src/configgen.py -maph ${CMAKE_SOURCE_DIR}/src/config.xml > ${GENERATED_SRC}/configvalues.h
-    DEPENDS ${CMAKE_SOURCE_DIR}/src/config.xml ${CMAKE_SOURCE_DIR}/src/configgen.py
-    OUTPUT ${GENERATED_SRC}/configvalues.h
-)
-set_source_files_properties(${GENERATED_SRC}/configvalues.h PROPERTIES GENERATED 1)
 
 include_directories(${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/qtools ${GENERATED_SRC})
 add_library(vhdlparser STATIC
-${GENERATED_SRC}/configvalues.h
 CharStream.cc
 ParseException.cc
 Token.cc
@@ -16,4 +8,7 @@ TokenMgrError.cc
 VhdlParser.cc
 VhdlParserTokenManager.cc
 VhdlParserIF.cpp
+)
+add_dependencies(vhdlparser
+    generate_configvalues_header
 )
